### PR TITLE
remove unused "use" of AsciiExt

### PR DESCRIPTION
**Pull request by bltnkl**
As of 1.23 stable, 
one now longer needs to import std::ascii::AsciiExt. See changelog for 1.23.
Also fixed few formatting issues.

### CODE REVIEW

File: src/bin/mtxt.rs [hunk 1]
```diff
@@ -3,15 +3,14 @@
...
-use std::ascii::AsciiExt;
 use std::env::args;
-use std::io::{self, Write, Read};
+use std::io::{self, Read, Write};
 use std::process::exit;
 
 use extra::option::OptionalExt;
 use extra::io::{fail, WriteExt};
 
-static HELP: &'static str = /* @MANSTART{mtxt} */ r#"
```
Comment (jackpot51) [inline src/bin/mtxt.rs]: MANSTART{mtxt} should not have been removed.

**Comment by bltnkl** [comment]
Added /* @MANSTART{mtxt} */

**Comment by jackpot51** [comment]
Thanks!

**Comment by bltnkl** [comment]
Please, close https://github.com/redox-os/extrautils/issues/13

**Comment by jackpot51** [comment]
Done!

**Pull request merged**